### ISSUE TITLE
ActiveSync: UNIQUE constraint failed: folder account/path

### ISSUE THREAD

**Issue by benbucksch** [mustang-im/mustang#204]
### Reproduction
1. Set up neil@beonex.on...
2. Select ActiveSync as protocol
3. Finish setup
4. Click on the account and inbox

### Actual result
* UNIQUE constraint failed: folder.accountID, folder.path

![image](https://github.com/user-attachments/assets/f3c3e1b4-e3c0-4b82-8ec0-ea6cf6b0d896)

### Expected result
* No error

**Comment by benbucksch** [comment]
This SQL request failed:
```
{
    "sourceParts": [
        "\n        INSERT INTO folder(\n          accountID, name, path,\n          parent, specialUse\n        ) VALUES(\n          ",
        ", ",
        ", ",
        ",\n          ",
        ", ",
        "\n        )"
    ],
    "parameters": [
        1,
        "Sent Items",
        "25",
        null,
        "sent"
    ]
}
```

**Comment by benbucksch** [comment]
Fixed by #207
